src/locales/en/translation.json

```diff
@@ -406,8 +406,8 @@
         "forgotPasswordDescription": "ROSE Wallet does not store your password and cannot help you retrieve it. If you forgot your password, you can delete your locked profile here. After that, you can create a new one using your mnemonic phrase or private keys, and use your ROSE tokens again.",
         "title": "Delete Profile"
       },
-      "description": "Log into your existing user profile on this computer to access the wallets you already added.",
-      "enterPasswordHere": "Enter your password here",
+      "description": "Enter your password to access your existing wallets on this device.",
```

Comment:
Maybe further simplification to something like "Unlock your profile/wallet by providing password.". I don't particularly like "existing wallets" wording.

Reply:
cc @donouwens if you want to iterate more on it

(but I think we did want to hint that it's only stored on this device)

old extension says: "Please enter your password to access your ROSE tokens"